app/admin/book.rb

```diff
@@ -4,13 +4,41 @@
     column :title
     column :author
     column :published_date
-    column :price_cents
-    column :discount_cents
+    column(:price) { |book| number_to_currency book.price }
+    column(:discount) { |book| number_to_currency book.discount }
     column :category
     column :description
     actions
   end
 
+  show do
+    attributes_table do
+      row :id
+      row :title
+      row :published_date
+      row :author
+      row :category
+      row :description
+      row(:price) { |book| number_to_currency book.price }
+      row(:discount) { |book| number_to_currency book.discount }
+      row :order_count
+    end
+  end
+
+  form do |form|
```

Comment:
Unused block argument - `form`. You can omit the argument if you don't care about it.